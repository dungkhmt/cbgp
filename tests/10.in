20 20
20 36
20 11
11 2
2 3
3 9
9 12
12 8
8 20
5 13
13 4
4 16
16 15
15 7
7 5
6 19
19 1
1 14
14 18
18 6
18 7
6 1
17 10
15 9
10 14
12 15
16 9
13 19
5 8
20 2
5 6
17 14
16 3
12 7
10 18
4 2
20 13
4 1
1
0 0 20 2 21 3 22 0 11 3 8 2 18 10 16 6 22 4 14 14 17 2 18 8 13 1 12 12 19 9 28 0 14 13 17 11 9 1 18 4 
