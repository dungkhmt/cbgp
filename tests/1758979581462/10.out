(19, 10), (10, 1), (1, 2), (2, 8), (8, 11), (11, 7), (7, 19), (4, 12), (12, 3), (3, 15), (15, 14), (14, 6), (6, 4), (5, 18), (18, 0), (0, 13), (13, 17), (17, 5), (17, 6), (5, 0), (16, 9), (14, 8), (9, 13), (11, 14), (15, 8), (12, 18), (4, 7), (19, 1), (4, 5), (16, 13), (15, 2), (11, 6), (9, 17), (3, 1), (19, 12), (3, 0), 
numberIntersection=0 
minAngle=0.32175055439664213 
Var(edgeLength)=242.82867713426913 
minDistance=0.31622776601683794 
Var(nodePos)=690.9175 
sumAngle=10.097555161177294 
sumDistance=18157.720606482788 
minEdgeLength=1.0 
sumEdgeLength=687.9535869053586 

0.0000000000 0.3217505544 242.8286771343 0.3162277660 690.9175000000 10.0975551612 18157.7206064828 1.0000000000 687.9535869054

0: (2, 0),
1: (37, 5),
2: (36, 11),
3: (42, 0),
4: (7, 5),
5: (6, 4),
6: (31, 53),
7: (31, 29),
8: (42, 19),
9: (27, 53),
10: (31, 4),
11: (30, 44),
12: (15, 9),
13: (25, 53),
14: (38, 60),
15: (66, 1),
16: (27, 52),
17: (28, 50),
18: (14, 4),
19: (28, 22),

Time taken: 34.544 s
2 0 37 5 36 11 42 0 7 5 6 4 31 53 31 29 42 19 27 53 31 4 30 44 15 9 25 53 38 60 66 1 27 52 28 50 14 4 28 22 
0: (2, 0), 1: (37, 5), 2: (36, 11), 3: (42, 0), 4: (7, 5), 5: (6, 4), 6: (31, 53), 7: (31, 29), 8: (42, 19), 9: (27, 53), 10: (31, 4), 11: (30, 44), 12: (15, 9), 13: (25, 53), 14: (38, 60), 15: (66, 1), 16: (27, 52), 17: (28, 50), 18: (14, 4), 19: (28, 22), 
numberIntersection=0 minAngle=0.32175055439664213 sumAngle=10.097555161179617 minEdgeLength=1.0 sumEdgeLength=687.953586905361 minDistance=0.31622776601683794 sumDistance=18157.72060648272 
